samples/TagHelperSample.Web/Controllers/HomeController.cs

```diff
@@ -76,5 +76,10 @@ public IActionResult Edit(int id, User user)
             ViewBag.Items = _items;
             return View();
         }
+
+        public IActionResult ConditionalComment()
+        {
+            return View();
+        }
```

Comment:
Seems like a better fit to create a new controllers. This doesn't really jive with the other actions here.